packages/ord-connect/src/components/SelectWalletModal/index.tsx

```diff
@@ -396,6 +411,107 @@ export function SelectWalletModal({
     };
   }, [wallet, onConnectUnisatWallet, disconnectWallet]);
 
+  const orderedWalletList = useMemo<WalletListItemProp[]>(() => {
+    const walletList: WalletListItemProp[] = [
+      {
+        wallet: Wallet.OKX,
+        subtitle: "Available on OKX app",
+        onConnect: onConnectOKXWallet,
+        icon: OKXWalletIcon,
+        setErrorMessage,
+        isMobileDevice: isMobile,
+        renderAvatar,
+        isAvailable: !isMobile || (isMobile && network === Network.MAINNET),
+      },
+      {
+        wallet: Wallet.UNISAT,
+        subtitle: "Coming soon on mobile browsing",
+        onConnect: onConnectUnisatWallet,
+        icon: UnisatWalletIcon,
+        setErrorMessage,
+        isMobileDevice: isMobile,
+        renderAvatar,
+        isAvailable: !isMobile,
+      },
+      {
+        wallet: Wallet.XVERSE,
+        subtitle: "Available on Xverse app",
+        onConnect: onConnectXverseWallet,
+        icon: XverseWalletIcon,
+        setErrorMessage,
+        isMobileDevice: isMobile,
+        renderAvatar,
+        isAvailable: true,
+      },
+      {
+        wallet: Wallet.MAGICEDEN,
+        subtitle: "Coming soon on mobile browsing",
+        onConnect: onConnectMagicEdenWallet,
+        icon: MagicEdenWalletIcon,
+        setErrorMessage,
+        isDisabled: isMobile,
+        isMobileDevice: isMobile,
+        renderAvatar,
+        isAvailable: !isMobile,
+      },
+      {
+        wallet: Wallet.LEATHER,
+        subtitle: "Coming soon on mobile browsing",
+        onConnect: onConnectLeatherWallet,
+        icon: LeatherWalletIcon,
+        setErrorMessage,
+        isDisabled: isMobile,
+        isMobileDevice: isMobile,
+        renderAvatar,
+        isAvailable: !isMobile,
+      },
+    ];
+
+    if (!walletsOrder) {
+      return walletList;
+    }
+
+    const newList = walletsOrder.reduce<WalletListItemProp[]>(
```

Comment:
Wait I think this logic can be simpler.

1. We just pass the wallet (e.g Wallet.OKX, Wallet.Leather etc.)
2. Then we just do sort of walletList based on the walletOrder. Something like walletList.sort((a, b) => walletsOrder.includes(a)). Need to check the sorting logic I just typed it now haha